python/example/ou_lif/Readme.md

```diff
@@ -0,0 +1,37 @@
+# Ornstein-Uhlenbeck input to LIF neuron
+
+This example is taken from [Jannik Luboeinski's repo](https://github.com/jlubo/arbor_ou_lif_example)
+and adapted for inclusion into Arbor.
+
+The simulation consists of a single compartment cable cell with simple leaky integrate-and-fire (LIF)
+dynamics with Ornstein-Uhlenbeck (OU) input current.  The OU input current can approximate the
+synaptic input from a population of neurons.  This is mathematically based on the diffusion
+approximation, which analytically describes the input current that arises from presynaptic Poisson
+spiking if the spiking activity is sufficiently high and the evoked postsynaptic potentials are
+sufficiently small compared to the threshold potential [1,2].  Moreover, the exponentially decaying
+autocorrelation function of the OU process resembles that of currents evoked by the spiking activity
+of many neocortical neurons [3].
+
+Here, we use the Ornstein-Uhlenbeck process in the following form, `dI/dt = -(1/τ) * (I - μ) +
+√(2/τ) * σ * W`.  Assuming homogeneous Poisson spiking for a population of *N* neurons in the
+putative population, the mean current is `μ = N * f * w_out`, and the volatility (standard
```

Comment:
```suggestion
√(2*τ) * σ * W`.  Assuming homogeneous Poisson spiking at frequency *f* for a putative population of *N* neurons, 
the mean current is `μ = N * f * w_out`, and the volatility (standard
```
The multiplication is just a suggestion: in my opinion, one can follow the equations more easily this way (also see the other related comments).